ui/src/views/offering/AddComputeOffering.vue

```diff
@@ -340,7 +340,7 @@
         </a-form-item>
         <a-form-item name="computeonly" ref="computeonly">
           <template #label>
-            {{ $t('label.computeonly.offering') }}
+            <tooltip-label :title="$t('label.computeonly.offering')" tooltip="Option to specify root disk related information in the compute offering or to link a disk offering to the compute offering"/>
```

Comment:
```suggestion
            <tooltip-label :title="$t('label.computeonly.offering')" tooltip="Option to directly map root disk related information to the compute offering"/>
```

cc @harikrishna-patnala please advise

Reply:
@sureshanaparti, I think it is better to mention both cases.

how about "Option to map root disk related information directly with a disk offering". If we say this we might have to change the toggle button from "Compute only offering" to "Map to a disk offering" and keep it off by default.

Reply:
ok @harikrishna-patnala if both cases makes sense for "Compute only offering", keep that.